### ISSUE TITLE
template answers is posible?

### ISSUE THREAD

**Issue by NickyDark1** [NVIDIA/logits-processor-zoo#7]
you may be able to answer

```py
<reasoning>
...
</reasoning>
<answer>
...
</answer>
```

answer where the '...' are?

**Comment by aerdem4** [comment]
I may suggest you to:

- Start your prompt with <reasoning>
- Use ForceLastPhraseLogitsProcessor("</reasoning><answer>")

This may work depends on your LLM and use case.
